source/scss/_pf-shame.scss

```diff
@@ -1,4 +1,19 @@
-/*!
- * Patternfly Shame
- * The nest fot all our shamefull code.
- */
+/*
+Copyright 2016 Red Hat, Inc
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//
+// Patternfly Shame
+// --------------------------------------------------
+// The nest fot all our shamefull code.
```

Comment:
Just more types: "The nest ~~fot~~ **for** all our ~~shamefull~~ **shameful** code."